frontend/src/state/reducers/studentReducer.ts

```diff
@@ -50,20 +53,25 @@ export const studentReducer = (
           return draft;
         }
 
-        if (draft.student.completedCourses === undefined) {
```

Comment:
Is deleting this intentional?

Reply:
oh because completedCourses will never be undefined i think

Reply:
okay i trust u hehe 👍

Reply:
HAHA i dont trust myself maybe we can add that back later